velox/exec/Driver.cpp

```diff
@@ -340,22 +340,62 @@ void Driver::enqueueInternal() {
 
 // Call an Oprator method. record silenced throws, but not a query
 // terminating throw. Annotate exceptions with Operator info.
-#define CALL_OPERATOR(call, operator, methodName)                       \
+#define CALL_OPERATOR(call, op, opIdx, methodEnum)                      \
   try {                                                                 \
     threadNumVeloxThrow() = 0;                                          \
+    opCallStatus_.timeStartMs = getCurrentTimeMs();                     \
+    opCallStatus_.opId = opIdx;                                         \
```

Comment:
it is a bit strange to see opId = opIdx... Can we change op and opIdx to operator and operatorId?

Would it make sense to write this as : opCallStatus_.start(operatorId, methodEnum) and opCallStatus_.stop()?

Reply:
@mbasmanova 
'operator' cannot be used - it is a special word in c++.
We have already a mix up of operator id and index, which is the same.
For example: curOpIndex_.
I just recently found out that operatorId is literally index.

What do we want here, because options are many and would be good to reduce 'back & forth'.